artifacts-guidance.md

```diff
@@ -0,0 +1,5 @@
+# Guidance for Artifacts Authors
+
+Content other than OCI container images MAY be packaged using the image manifest.
+When this is done, the `config.mediaType` value MUST be set to a value specific to the artifact type or the [empty value](manifest.md#guidance-for-an-empty-descriptor).
```

Comment:
If we are targetting 1.1 I think we should be also be specifying the artifactType field take precedence over config mediaType when present.